src/main/webapp/components/workspace/workspace.js

```diff
@@ -89,6 +89,7 @@ export default () => {
   const [currentQuery, setCurrentQuery] = useState(null)
   const [lists, setLists] = React.useState(null)
   const [queries, setQueries] = useState()
+  const [resultsToExport, setResultsToExport] = React.useState([])
```

Comment:
I think we should consider having this state managed by context instead of passing it down through props. Similar to how the [selection interface](https://github.com/connexta/revelio/blob/master/src/main/webapp/react-hooks/use-selection-interface.js) works. @samuelechu thoughts?

Reply:
See above comment